xbmc/GUIInfoManager.cpp

```diff
@@ -1277,377 +1763,709 @@ const infomap network_labels[] = {{ "isdhcp",            NETWORK_IS_DHCP },
-///     playing item and "MusicPlayer.Position(number).Genre" position is
-///     relative to the start of the playlist
+///     @return The genre(s) of current song.
+///     <p>
+///   }
+///   \table_row3{   <b>`MusicPlayer.offset(number).Genre`</b>,
+///                  \anchor MusicPlayer_OffSet_Genre
+///                  _string_,
+///     @return The genre(s) of the song with an offset `number` with respect
+///     to the current playing song.
+///     @param number - the offset song number with respect to the current playing
+///     song.
+///     <p>
+///   }
+///   \table_row3{   <b>`MusicPlayer.Position(number).Genre`</b>,
+///                  \anchor MusicPlayer_Position_Genre
+///                  _string_,
+///     @return The genre(s) of the song with an offset `number` with respect
+///     to the start of the playlist.
+///     @param number - the offset song number with respect to the start of the
+///     playlist
+///     song.
+///     <p>
 ///   }
 ///   \table_row3{   <b>`MusicPlayer.Lyrics`</b>,
 ///                  \anchor MusicPlayer_Lyrics
 ///                  _string_,
-///     Lyrics of current song stored in ID tag info
+///     @return The lyrics of current song stored in ID tag info.
+///     <p>
 ///   }
 ///   \table_row3{   <b>`MusicPlayer.Year`</b>,
 ///                  \anchor MusicPlayer_Year
 ///                  _string_,
-///     Year of release of current song\, also available are
-///     "MusicPlayer.offset(number).Year" offset is relative to the current
-///     playing item and "MusicPlayer.Position(number).Year" position is
-///     relative to the start of the playlist
+///     @return The year of release of current song.
+///     <p>
+///   }
+///   \table_row3{   <b>`MusicPlayer.offset(number).Year`</b>,
+///                  \anchor MusicPlayer_Offset_Year
+///                  _string_,
+///     @return The year of release of the song with an offset `number` with
+///     respect to the current playing song.
+///     @param number - the offset numbet with respect to the current song.
+///     <p>
+///   }
+///   \table_row3{   <b>`MusicPlayer.Position(number).Year`</b>,
+///                  \anchor MusicPlayer_Position_Year
+///                  _string_,
+///     @return The year of release of the song with an offset `number` with
+///     respect to the start of the playlist.
+///     @param number - the offset numbet with respect to the start of the
+///     playlist.
+///     <p>
 ///   }
 ///   \table_row3{   <b>`MusicPlayer.Rating`</b>,
 ///                  \anchor MusicPlayer_Rating
 ///                  _string_,
-///     Numeric Rating of current song\, also available are
-///     "MusicPlayer.offset(number).Rating" offset is relative to the current
-///     playing item and "MusicPlayer.Position(number).Rating" position is
-///     relative to the start of the playlist
+///     @return The numeric Rating of current song.
```

Comment:
Possibly helpful to add that rating is out of 10.
True for all ratings and user rating